submissions/chernetskyi8704/OOP-Exercise/index.js

```diff
@@ -0,0 +1,42 @@
+const dog = {
+  species: "dog",
+  legs: 4,
+  hands: 0,
+  name: "Molly",
+  gender: "female",
+  saying: "WOOF-WOOF!",
+};
+
+const cat = {
+  species: "cat",
+  legs: 4,
+  hands: 0,
+  name: "Whiskey",
+  gender: "male",
+  saying: "Meow!",
+};
+
+const woman = {
+  species: "human",
+  legs: 2,
+  hands: 2,
+  name: "Kate",
+  gender: "female",
+  saying: "Have a good day!",
+};
+
+const man = {
+  species: "human",
+  legs: 2,
+  hands: 2,
+  name: "Dimitrij",
+  gender: "male",
+  saying: "Nice to meet you!",
+};
+
+const allInhabitants = [dog, cat, woman, man];
+const keys = ["species", "legs", "hands", "name", "gender", "saying"];
```

Comment:
`key` is a generic term. Inhabitants are charaterized rather by features or properties.
Naming is important.